moto/s3/models.py

```diff
@@ -59,7 +59,9 @@
     "GLACIER",
     "DEEP_ARCHIVE",
 ]
-DEFAULT_KEY_BUFFER_SIZE = 16 * 1024 * 1024
+DEFAULT_KEY_BUFFER_SIZE = int(
```

Comment:
This should be set in moto/settings.py, to ensure we have all env variables in one place.
Can  you make it a method, as well? As a variable, it can only be set once - with a method, people can set different values for the env variable if and when required, and it will always get the most recent env variable value.

Reply:
Sounds good, I will make that update.